server/scripts/fhirDefinitions.ts

```diff
@@ -0,0 +1,77 @@
+import * as fs from 'fs'
+import { join } from 'path'
+
+import { Photon, StructureDefinitionCreateInput } from '@prisma/photon'
+
+import { HL7_AUTHOR } from '../src/constants'
+
+const photon = new Photon()
+
+const parseBundles = async (directory: string) => {
+  const dir = await fs.promises.opendir(directory)
```

Comment:
Maybe we should check at some point that we process only the json files we want to process.
Or maybe we can add a subfolder in server/fhir, just in case we need to have something else in this folder in the future

Reply:
good remark, now you specify a list of files to parse instead of a folder